### DISCUSSION TITLE
RazBerry 2 green LED almost always on

### DISCUSSION THREAD

**Discussion by AlfaBravoX** [zwave-js/zwave-js-ui#3239] [Q&A]
Hi,

I'd appreciate help. I have search a lot but did not find an answer. I have following setup:

Razberry GPIO hat with RPI3 
Raspbian Bullseye 11.7
docker
Danfoss LC-13 TRV v 1.1 (not zwave+)

All seems to work well, however the green LED on hat is almost always on, with some exceptions when it powers OFF for a moment time to time otherwise stays ON. Anyone experienced same behaviour.

Also it appears that ZUI shows different FW version then original `ZMESerialUpdater` shows:

```
SERIAL INFO 
		 VERSION:  05 25  VENDOR:  01 47
```

whereas ZUI shows:

<img width="204" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/11148591/d494c5ca-1f44-48cb-8d8e-4dc5b2ed0277">

any ideas?

**Comment by kpine** [comment]
> All seems to work well, however the green LED on hat is almost always on, with some exceptions when it powers OFF for a moment time to time otherwise stays ON. Anyone experienced same behaviour.

Have you tried contacting Z-Wave.me support about this? Maybe it's explained in their manual? These sorts of things are vendor specific, like the Aeotec LED.

> Also it appears that ZUI shows different FW version then original `ZMESerialUpdater` shows:
> 
> ```
> SERIAL INFO 
> 		 VERSION:  05 25  VENDOR:  01 47
> ```

01 47 vendor ID is in hex format (0x0147). So I would suspect 05 25 is 0x5 0x25, or 5 37 in decimal, same as Z-Wave JS.

**Reply by AlfaBravoX** [reply]
Ok, thanks for pointing to hex format, this seems to be the case. Regarding TX green LED I will turn to Z-Wave.me support..
